src/compas/scene/scene.py

```diff
@@ -26,9 +28,26 @@ class SceneObjectNode(TreeNode):
     """
 
     def __init__(self, sceneobject):
-        super(SceneObjectNode, self).__init__(name=sceneobject.name)
+        super(SceneObjectNode, self).__init__()
         self.object = sceneobject
 
+    @property
+    def data(self):
+        return {
+            "item": str(self.object.item.guid),
+            "settings": self.object.settings,
+            "children": [child.data for child in self.children],
+        }
+
+    @classmethod
+    def from_data(cls, data):
+        raise TypeError("SceneObjectNode cannot be created from data. Use SceneTree.from_data instead.")
+
+    @property
+    def name(self):
+        if self.object:
```

Comment:
and if no `self.object` nothing happens? no error either?

Reply:
@Licini can you still check if that is indeed the intended behaviour?